File: iniciante/javascript/1048.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function calcularSalario(a){
    let salario
    let reajuste
    let percentual

    if(a >= 0  && a <= 400.00){
        salario = a + 0.15 * a
        reajuste = 0.15 * a
        percentual = '15 %'
    }
    if(a >= 400.01  && a <= 800.00){
        salario = a + 0.12 * a
        reajuste = 0.12 * a
        percentual = '12 %'
    }
    if(a >= 800.01  && a <= 1200.00){
        salario = a + 0.10 * a
        reajuste = 0.10 * a
        percentual = '10 %'
    }
    if(a >= 1200.01  && a <= 2000.00){
        salario = a + 0.07 * a
        reajuste = 0.07 * a
        percentual = '7 %'
    }
    if(a > 2000.00){
        salario = a + 0.04 * a
        reajuste = 0.04 * a
        percentual = '4 %'
    }

    return `Novo salario: ${salario.toFixed(2)}\nReajuste ganho: ${reajuste.toFixed(2)}\nEm percentual: ${percentual}`
}

let entrada = parseFloat(lines.shift())

console.log(calcularSalario(entrada))

File: iniciante/python/1048.py
# -*- coding: utf-8 -*-

entrada = float(input())

novo_salario = 0.0
reajuste = 0.0
percentual = 0

if entrada >= 0 and entrada <= 400.00:
    novo_salario = entrada + entrada * 0.15
    reajuste = entrada * 0.15
    percentual = 15
elif entrada >= 400.01 and entrada <= 800.00:
    novo_salario = entrada + entrada * 0.15
    reajuste = entrada * 0.12
    percentual = 12
elif entrada >= 800.01 and entrada <= 1200.00:
    novo_salario = entrada + entrada * 0.10
    reajuste = entrada * 0.10
    percentual = 10
elif entrada >= 1200.01 and entrada <= 2000.00:
    novo_salario = entrada + entrada * 0.07
    reajuste = entrada * 0.07
    percentual = 7
else:
    novo_salario = entrada + entrada * 0.04
    reajuste = entrada * 0.04
    percentual = 4

print(f'Novo salario: {novo_salario:.2f}')
print(f'Reajuste ganho: {reajuste:.2f}')
print(f'Em percentual: {percentual} %')
